python/iceberg/api/transforms/transform_util.py

```diff
@@ -59,7 +60,7 @@ def human_hour(hour_ordinal):
 
     @staticmethod
     def base_64_encode(buffer):
-        raise NotImplementedError()
```

Comment:
similar to the other PR this should probably be a set of functions rather than static methods in a class

Reply:
Updated.